recipes-connectivity/prplmesh/prplmesh.bb

```
@@ -10,25 +10,43 @@ OECMAKE_SOURCEPATH = "${S}/"
 
 SECTION = "net"
 LICENSE = "BSD-2-Clause-Patent.txt"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
-DEPENDS = "python-native python-pyyaml-native json-c openssl zeromq"
+# LIC_FILES_CHKSUM = "file://LICENSE;md5=7b45146d47e73bcbac1068e5cfc2a9fb"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=0518d409dae93098cca8dfa932f3ab1b"
```

I guess you changed the LICENSES_PATH because the LICENSE file is in the top directory, not in the LICENSES subdirectory. So I think this should be
```suggestion
LIC_FILES_CHKSUM = "file://BSD-2-Clause-Patent.txt;md5=0518d409dae93098cca8dfa932f3ab1b"
```

---

This line is about license file in the https://github.com/prplfoundation/prplMesh/tree/d9eb170b69c1c225e40df5c191e8978568ec2e08 repository, not about reference license file in `LICENSES_PATH`

---

Solved in #19